.github/workflows/pip-compile.yml

```
@@ -0,0 +1,77 @@
+---
+name: "Refresh pinned dependencies"
+
+"on":
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+  push:
+    branches:
+      - devel
+    paths:
+      - .github/workflows/pip-compile.yml
+      - "tests/*.in"
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  refresh:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: "devel"
+      - name: Fetch required contents of ansible-core
+        run: |
+          python docs/bin/clone-core.py
+      - name: Set up nox
+        uses: wntrblm/nox@2023.04.22
+        with:
+          python-versions: "3.9"
+      - name: Use a branch named pip-compile
+        id: branch
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+
```

This is an incorrect email for crediting commits to the GHA user. Here's the right one: https://github.com/fregante/setup-git-user/blob/77c1b55/action.yml#L14.
```suggestion
      - name: Set up Git to credit commits to GitHub Actions
        uses: fregante/setup-git-user@v2
      - name: Use a branch named pip-compile
        id: branch
        run: |
```

---

I'm not going to pull in a dependency for two trivial lines of code, but I will change the `user.email`.